lib/src/util/windows/wmi.cc

```diff
@@ -49,7 +49,7 @@ namespace facter { namespace util { namespace windows {
         if (FAILED(hres)) {
             throw wmi_exception(format_hresult("failed to create IWbemLocator object", hres));
         }
-        _pLoc = scoped_resource<IWbemLocator *>(move(pLoc),
+        _pLoc = scoped_resource<IWbemLocator *>(pLoc,
```

Comment:
Do we want to remove `move`? In this case `pLoc` is an actual pointer, whereas `HANDLE` is an opaque index into a table of pointers.